lib/core/src/lib/card-view/components/card-view-selectitem/card-view-selectitem.component.ts

```diff
@@ -71,10 +71,14 @@ export class CardViewSelectItemComponent extends BaseCardView<CardViewSelectItem
         this.filter$.next(value.toString());
     }
 
-    isEditable(): boolean {
+    get isEditable(): boolean {
         return this.editable && this.property.editable;
     }
 
+    get notClickable(): boolean {
```

Comment:
the name does not reflect anything? why it's called "not clickable"? you can't click it? all I see is the custom label color attached to this property, and that property does not reflect the "not clickable" as well